andela/ah-orcas
# #161254651 Integrate Travis CI with README badge

#### What does this PR do?
* This PR integrates Travis CI badge with README badge.
#### Description of Task to be completed?
* Add Travis badge to the README file
* Create a .travis.yml file in the project folder

#### How should this be manually tested?
* Open the repo : git clone https://github.com/andela/ah-orcas.git
* switch to integrate-Travis branch
* Check the README file

#### Any background context you want to provide?
* Travis makes working in a team for a software project easier with automated builds which  are triggered automatically when each developer checks in their code to the repository.
* It also helps  to automatically run tests and other preliminary build steps

#### What are the relevant pivotal tracker stories?
[#161254651](https://www.pivotaltracker.com/story/show/161254651)
#### Screenshots
<img width="986" alt="screen shot 2018-10-23 at 12 29 04" src="https://user-images.githubusercontent.com/25586421/47350840-74b64500-d6bf-11e8-8f0a-b5955f0fb964.png">
